src/cli/List.cpp

```diff
@@ -50,6 +50,7 @@ int List::execute(const QStringList& arguments)
                                QObject::tr("Key file of the database."),
                                QObject::tr("path"));
     parser.addOption(keyFile);
+    parser.addOption({{"R", "recursive"}, "Recursive mode, list elements recursively"});
```

Comment:
This needs an explicit `QStringList()` lvalue as in the example above. The description also needs to be wrapped in `QObject::tr()`.